lib/ProductOpener/Products.pm

```diff
@@ -1926,14 +1926,16 @@ sub index_product($)
 
 	my %keywords;
 
+	my $product_lc = $product_ref->{lc} ||$lc;
+
 	foreach my $field (@string_fields, @tag_fields) {
 		if (defined $product_ref->{$field}) {
-			foreach my $tag (split(/,|'|\s/, $product_ref->{$field} )) {
+			foreach my $tag (split(/,|'|’|\s/, $product_ref->{$field} )) {
 				if (($field eq 'categories') or ($field eq 'labels') or ($field eq 'origins')) {
 					$tag =~ s/^\w\w://;
 				}
```

Comment:
That's actually a right single quotation mark. Would it make sense to only do this if there's no left single quotation mark? I'm not really sure why a quotation mark should be used in place of an apostrophe.

Reply:
Word processors like LibreOffice convert L'été to L’été. The use of the right quotation mark as an apostrophe seems widespread, in fact it seems to be the unicode standard

https://tedclancy.wordpress.com/2015/06/03/which-unicode-character-should-represent-the-english-apostrophe-and-why-the-unicode-committee-is-very-wrong/

Reply:
Anyway, the goal of this is to make sure that in search, we remove articles so that queries like "ail" (garlic) match "crème à l'ail" (garlic cream). So even if it an actual quotation mark, it will be removed anyway.

Reply:
Sounds like the consortium's guideline is controversial to say the least, but let's roll with it. 🤷‍♂️